rules/rbe_repo.bzl

```diff
@@ -845,6 +856,9 @@ def rbe_autoconfig(
       digest: Optional. The digest of the image to pull.
           Should not be set if 'tag' is used.
           Must be set together with 'registry' and 'repository'.
+      detect_java_home: Optional. Default False. Should only be set
```

Comment:
If java_home is not specified, and detect_java_home is false, then the java_home in java_runtime() will be empty? or the java config will not be create at all?

Reply:
Saw the fallback logic below. But that's kinda complicated...

Reply:
added notes about fallback logic. ptal